### PR TITLE
Changing blog index layout to include excerpts, rather than whole posts

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- Changed blog index layout Liquid template.

screen shot 2017-05-04 at 2 04 39 pm

## Type of Pull Request:

- [ ] Blog Post
- [ ] Content (Documentation updates or typo-fixes)
- [X] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125235](https://oktainc.atlassian.net/browse/OKTA-125235)

## How Has This Been Tested?
- [ ] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested the change across Chrome, Firefox, Safari, and mobile.

## Primary Reviewers:

- *Tag reviewer(s)*

- [Blog Post Google Doc](https://docs.google.com)

## Before Merging:

- [ ] I have had this PR reviewed and approved on GitHub.
- [ ] I have squashed my commits into `1` commit or will use the **Squash and Merge button**.

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Someone with more technical chops than me should review this. I suggested Jordan but anyone on the team can review.

File: _source/_layouts/blog_index.html [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
+        {% for post in site.posts %}
+            {% capture _post_content %}
+                {{ post.content | strip_html | truncatewords:75 }}
+                <a href="{{post.url}}">Read more</a>
+            {% endcapture %}
+            
+            {% include blog_post.html post_content=_post_content %}
+            <br/>
+
+        {% endfor %}
```
Comment (mraible) [inline _source/_layouts/blog_index.html]: This causes some strange issues - as reported by @jmelberg-okta. Putting everything on one line seems to solve the problem.

```
        {% for post in site.posts %}
            {% capture _post_content %}{{ post.content | strip_html | truncatewords:75 }} <a href="{{post.url}}">Read more</a>{% endcapture %}
            {% include blog_post.html post_content=_post_content %}
        {% endfor %}
```

Comment (rchild-okta) [inline _source/_layouts/blog_index.html]: Yep, looks like this works. Am curious about whether we can use Jekyll's [excerpt](https://jekyllrb.com/docs/posts/#post-excerpts) feature. Tried it out and it doesn't capture the heading markup that you have, which does feel necessary.

After updating with @mraible's fix, you'll need to rebase against latest master to pull in the TravisCi changes to be able to merge.

@vijetmahabaleshwar-okta this is also a good one to add a test for.

Comment (jakubvul-okta) [inline _source/_layouts/blog_index.html]: @rchild-okta I tried using Excerpt, but there wasn't a graceful way to add the Read Me there, at least that I could find.

**Pull request merged**